libs/dh/shared/feature-authorization/src/lib/dh-selected-actor.component.scss

```
@@ -14,18 +14,84 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+.actors-dropup-container {
+  background-color: white;
```

suggestion: let's use a variable

```suggestion
  background-color: var(--watt-color-neutral-white);
```